apps/frontend/src/components/global/ExportXCCDFResults.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-tooltip top>
+    <template #activator="{on}">
+      <IconLinkItem
+        key="export_xccdf"
+        text="Export as XCCDF Results"
+        icon="mdi-xml"
+        @click="exportXCCDF()"
+        v-on="on"
+      />
+    </template>
+    <span>Export as XCCDF</span>
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+import IconLinkItem from '@/components/global/sidebaritems/IconLinkItem.vue';
+import {Filter, FilteredDataModule} from '@/store/data_filters';
+import {FromHDFToXCCDFMapper} from '@mitre/hdf-converters';
+import axios from 'axios';
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import {Prop} from 'vue-property-decorator';
+import {saveSingleOrMultipleFiles} from '../../utilities/export_util';
+export type FileData = {
+  filename: string;
+  data: string;
+};
+@Component({
+  components: {
+    IconLinkItem
+  }
+})
+export default class ExportXCCDF extends Vue {
+  @Prop({type: Object, required: true}) readonly filter!: Filter;
+  //exports .zip of XCCDFs if multiple are selected, if one is selected it will export that single file
+  exportXCCDF() {
+    axios.get<string>(`/static/export/xccdfTemplate.xml`).then(({data}) => {
+      const convertedFiles: {
+        filename: string;
+        data: string;
+      }[] = [];
+      const ids = FilteredDataModule.selected_file_ids;
+      for (const evaluation of FilteredDataModule.evaluations(ids)) {
+        const convertedData = new FromHDFToXCCDFMapper(
+          JSON.stringify(evaluation.data),
+          data
+        ).toXCCDF();
+        convertedFiles.push({
+          filename: evaluation.from_file.filename + '.xml',
+          data: convertedData
+        });
+      }
+      for (const evaluation of FilteredDataModule.profiles(ids)) {
```

Comment:
is it an intentional choice to support profile jsons for export?  @ejaronne

the reverse mapper does not support profile jsons itself so i suggest removing this bit of functionality entirely

Reply:
Both exports are supported now.